tests/base/test_server.py

```diff
@@ -144,6 +145,26 @@ def test_post_train(app, rasa_default_train_data):
     assert "error" in rjs
 
 
+@utilities.slowtest
+@pytest.inlineCallbacks
+def test_post_train_success(app, rasa_default_train_data):
+    model_config = {"pipeline": "keyword", "data": rasa_default_train_data}
+
+    response = app.post("http://dummy-uri/train?project=test&model=test",
+                        json=model_config)
+    time.sleep(3)
+    app.flush()
+    response = yield response
+    content = yield response.content()
+    assert response.code == 200
+    with io.open('./test_download.zip', 'wb') as f:
+        f.write(content)
+    import zipfile
```

Comment:
can we move it to the top of the method?

Reply:
do you mean the import statement?

Reply:
yes